code/modules/reagents/reagent_containers.dm

```diff
@@ -156,7 +156,8 @@
 		if(QDELETED(src))
 			return
 
-	reagents.clear_reagents()
+	if (!bartender_check(target)) // Needs to check for Bartender Slide before spilling.
+		reagents.clear_reagents() // Clear reagents of thrown container.
```

Comment:
```suggestion
```

Reply:
That seems like it'll break a bunch of shit. Are you sure? This instance of reagents.clear_reagents seems to be the main method of garbage collection for broken & thrown containers. At a glance, this probably would break beaker throw-splashing, too.